Sources/FDB/FDB/FDB+Errors.swift

```diff
@@ -279,6 +279,18 @@ public extension FDB {
             return result
         }
 
+        /// Indicates if current error is a native FoundationDB error
+        /// (i.e. errno is less than `8000`, after which custom FDBSwift errors start)
+        public var isNative: Bool {
+            self.errno < 8000
+        }
+
+        /// Indicates if current error is a custom FDBSwift error
+        /// (i.e. errno is greater than or equal to `8000`)
+        public var isCustom: Bool {
+            !self.isNative
+        }
```

Comment:
I don't think this is strictly necessary here, since the other case takes care of this situation.